en/docs/monitoring-and-insights/alerts-overview.md

```diff
@@ -0,0 +1,141 @@
+
+Configurable parameters
+
+| **Parameter** | **Description**                                                                 |
+|---------------|---------------------------------------------------------------------------------|
+| Metric        | CPU or Memory.                                                                  |
+| Threshold     | **mCPU** for CPU and **MiB** for Memory(e.g., 1000).                            |
+| Period        | Duration the threshold must be exceeded (e.g., 5 minutes).                      |
+
+!!! Tip
+    - **CPU**: mCPU (milliCPU) measures CPU usage in fractions of a core, where 1000m = 1 full core.
+    - **Memory**: MiB (Mebibyte) measures memory in binary units, where 1 MiB = 2^20^ bytes.
+
+### Log Alerts
+
+Log alerts trigger notifications when a specific phrase appears **a specific number of times** in your component logs within a defined time window. This helps to identify recurring issues or critical errors quickly, enabling faster troubleshooting.
+
+Configurable parameters
+
+| **Parameter** | **Description**                                                                 |
+|---------------|---------------------------------------------------------------------------------|
+| Search Phrase | Keyword or phrase to look for to trigger the alert (e.g., failed).                                               |
+| Count         | Minimum number of occurrences to trigger the alert (e.g., 10).                  |
+| Interval      | Time window for counting occurrences (e.g., 5 minutes).                         |
+
+### Build Failure Alerts
+
+Build failure alerts inform you if a build failure occurs for your component. This is essential for maintaining smooth development workflows.
+
+### Status Code Alerts
+
+Status code alert triggers when your component returns specific HTTP error(s) (e.g., **403** Forbidden, **500** Internal Error). These alerts help to detect issues affecting your component’s availability or speed.
+
+Configurable parameters
+
+| **Parameter** | **Description**                                                                 |
+|---------------|---------------------------------------------------------------------------------|
+| Status Code   | Error code or series (e.g., 400:Bad Request).                                   |
+| Count         | Minimum number of occurrences (e.g., 5).                                        |
+| Interval      | Time window (e.g., 5 minutes).                                                  |
+
+!!! note
+    Status code alerts are only supported for API proxy component types.
+
+## Configure Alert
+
+Follow these steps to configure an alert:
+
+1. Navigate to the component you wish to configure alerts for.
+
+    !!! info
+        you need to be a **Choreo DevOps** or **Choreo Platform Engineer** inorder to create Alerts.  
```

Comment:
Do we have a doc explaining how to assign these rules? It would be good to link it here